.github/workflows/build_tests.yml

```diff
@@ -9,36 +9,30 @@ on:
   workflow_dispatch:
 
 jobs:
-  build_tests:
+  py_build_tests:
+    uses: neongeckocom/.github/.github/workflows/python_build_tests.yml@master
+    with:
+      test_manifest: true
+      manifest_ignored: "test/**,scripts/**,CHANGELOG.md"
+  pip_audit:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: [ 3.7, 3.8, 3.9, "3.10" ]
+        python-version: [ "3.10" ]
```

Comment:
we should be testing all versions, this will miss dependency issues, eg, precise-lite plugin won't work in py 3.10

Reply:
not important in this specific skill, just a general comment re all latest PRs